src/NECompletion/NECContext.class.st

```diff
@@ -267,13 +268,23 @@ NECContext >> findSourceRangeFor: aNumber [
 										ifFalse: [^ nil]]]]]
 ]
 
+{ #category : #accessing }
+NECContext >> globals [
+	^ globals
+]
+
+{ #category : #accessing }
+NECContext >> globals: anObject [
+	globals := anObject
+]
+
 { #category : #'private-type-guessing' }
 NECContext >> guessArgument: aSHRange [ 
 	| name |
 	name := self sourceOf: aSHRange.
 	(name = 'html') ifTrue: [
-		 Smalltalk at: #WAHtmlCanvas ifPresent: [ :canvasClass | ^canvasClass].
-		 Smalltalk at: #WARenderCanvas ifPresent: [ :canvasClass | ^canvasClass]].
+		 globals at: #WAHtmlCanvas ifPresent: [ :canvasClass | ^canvasClass].
+		 globals at: #WARenderCanvas ifPresent: [ :canvasClass | ^canvasClass]].
```

Comment:
These should be completed removed, both lines

Reply:
If you remove them, please provide a hook to get back the behavior. Just removing it without any equivalent would reduce a lot the productivity in Seaside.

Reply:
that is outside the PR, if you want to do it, your are always welcome. 
If we have to keep the hack, it has to be in Seaside, not in the image.